swarm/network/stream/delivery.go

```diff
@@ -164,7 +164,7 @@ func (d *Delivery) handleChunkDeliveryMsg(ctx context.Context, sp *Peer, req int
 		mode = chunk.ModePutSync
 	}
 
-	log.Trace("handle.chunk.delivery", "ref", msg.Addr, "from peer", sp.ID())
+	log.Trace("handle.chunk.delivery", "ref", msg.Addr, "from peer", sp.bzzPeer.ID())
```

Comment:
Why is this necessary now, if we are not going to be refactoring all the peers in this PR?